src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.big-bar {
+  position: relative;
+  width: 100%;
+  height: 70px;
+  margin-top: 20px;
+  font-family: Avenir, sans-serif;
+}
+
+.big-image {
+  position: absolute;
+  z-index: 2;
+  margin-left: 26px;
+  top: 50%;
+  transform: translate(0, -50%);
+}
+
+.big-text-field {
+  position: absolute;
+  z-index: 1;
+  width: inherit;
+  height: inherit;
+  padding-left: 62px;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  color: #3d4e61;
+  font-size: 16px;
+  font-family: inherit;
+  font-weight: inherit;
+}
+
+.big-text-field:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
```

Comment:
that's property same as on small-field, you can create one class for both of your bars